src/Core/Constants.cs

```diff
@@ -28,6 +28,7 @@ public static class AuthenticationSchemes
 public static class FeatureFlagKeys
 {
     public const string SecretsManager = "secrets-manager";
+    public const string DisplayEuFlag = "display-eu-environment";
```

Comment:
⚠️ Given that the class has "keys" in it, I think this should be named `DisplayEuEnvironment`.